src/guides/v2.3/javascript-dev-guide/widgets/jquery-widgets-about.md

```diff
@@ -10,16 +10,17 @@ This guide discusses the following widgets:
 
 -  [Accordion widget]
 -  [Alert widget]
--  [Breadcrumb widget]
+-  [Breadcrumbs widget]
```

Comment:
Please, change the references at the bottom of the document correspondingly. It applies for all your modifications. 

Thank you!

Reply:
Fixed. Thanks, @rogyar!